vietanh2810/Sorting-Visualizer
# Feature setup vietanh

## Summary

This merge request resolves issue #4 by creating a display for the array which contains multiple bars of different height

## Changes Made

- add a SortingVisualizer dir with a jsx and a corresponding css file 
- In the Jsx file: render function return an array of bars, add others crucial functions

## Verification Steps

To verify that the login form is working correctly, follow these steps:

1. run npm start
2. go to http://localhost:3000/
3. Observe that lots of turquoise bars on the screen 

## Reviewers

@vietanh2810
